Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[workspace]
```

Comment:
It isn't great that we end up with overlapping workspace definitions and overlapping sets of `Cargo.lock` files. I suppose it wasn't possible to make it work with the existing workspaces under `rust/` and `scheds/rust/`? Hmm... there are no strong reasons to use separate workspaces for `rust/` and `scheds/rust/`, I don't think. If docs needs a single unified workspace, can you please make the build also use the top level workspace?

Reply:
> I don't think. If docs needs a single unified workspace, can you please make the build also use the top level workspace?

That works, and I think it also pushes things in a good direction overall (i.e. closer to the build being driven by cargo, and in a way that will "just work" w/ more folk's IDE's and rust tooling overall).